tensorzero-internal/src/clickhouse_migration_manager/migrations/migration_0014.rs

```diff
@@ -57,25 +57,6 @@ impl Migration for Migration0014<'_> {
             return Ok(true);
         }
 
-        let chat_inference_dataset_has_data =
-            table_is_nonempty(self.clickhouse, "ChatInferenceDataset", "0014").await?;
-        if chat_inference_dataset_has_data {
-            return Err(Error::new(ErrorDetails::ClickHouseMigration {
-                id: "0014".to_string(),
-                message: "ChatInferenceDataset has data. Your database state is invalid."
-                    .to_string(),
-            }));
-        }
-        let json_inference_dataset_has_data =
-            table_is_nonempty(self.clickhouse, "JsonInferenceDataset", "0014").await?;
-        if json_inference_dataset_has_data {
-            return Err(Error::new(ErrorDetails::ClickHouseMigration {
-                id: "0014".to_string(),
-                message: "JsonInferenceDataset has data. Your database state is invalid."
-                    .to_string(),
-            }));
-        }
-
```

Comment:
wait can we instead move this into the `apply()` function? I really do not want to drop tables with data in them.